src/components/scenes/SendScene.js

```diff
@@ -314,18 +315,19 @@ class SendComponent extends React.PureComponent<Props, State> {
         const displayAmount = bns.div(nativeAmount, cryptoDisplayDenomination.multiplier, UTILS.DIVIDE_PRECISION)
         const exchangeAmount = bns.div(nativeAmount, cryptoExchangeDenomination.multiplier, UTILS.DIVIDE_PRECISION)
         const fiatAmount = convertCurrencyFromExchangeRates(exchangeRates, selectedCurrencyCode, guiWallet.isoFiatCurrencyCode, parseFloat(exchangeAmount))
-        amountSyntax = `${displayAmount || '0'} ${cryptoDisplayDenomination.name} = (${fiatSymbol} ${fiatAmount.toFixed(2) || '0'})`
+        cryptoAmountSyntax = `${displayAmount || '0'} ${cryptoDisplayDenomination.name}`
+        if (fiatAmount) {
+          fiatAmountSyntax = `${fiatSymbol} ${fiatAmount.toFixed(2) || '0'}`
+        }
       } else {
-        amountSyntax = `${'0'} ${cryptoDisplayDenomination.name} = (${fiatSymbol} ${'0'})`
+        cryptoAmountSyntax = `0 ${cryptoDisplayDenomination.name}`
       }
 
       return (
-        <Tile
-          type={lockInputs ? 'static' : 'touchable'}
-          title={s.strings.fio_request_amount}
-          onPress={lockInputs ? undefined : this.handleFlipinputModal}
-          body={amountSyntax}
-        />
+        <Tile type={lockInputs ? 'static' : 'touchable'} title={s.strings.fio_request_amount} onPress={lockInputs ? undefined : this.handleFlipinputModal}>
+          <EdgeText style={{ fontSize: theme.rem(2) }}>{cryptoAmountSyntax}</EdgeText>
+          {!!fiatAmountSyntax && <EdgeText>{fiatAmountSyntax}</EdgeText>}
```

Comment:
I like a more explicit version like:

`fiatAmountSyntax == null ? null : <EdgeText>{fiatAmountSyntax}</EdgeText>`

But the version you have is fine if you want to keep it.